tests/test_api.py

```diff
@@ -0,0 +1,119 @@
+import os           # used to get the location of the testdata
+import pygeoj       # used to parse the geojson file
+import shapefile 
+import xarray as xr
+import sys
+
+__location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+
+###############################
+# Check file existence
```

Comment:
You can skip this. There is no need to check if the files exist, because the tests will fail if they don't, that's failure enough.

Reply:
Done